changelog/README.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 7th September 2023
+  * Added operation [Add rates](../operations/rates.md#add-rates)
+  * Adjusted [Rate](../operations/rates.md#rate) to be up to date.
```

Comment:
I think it's better to be explicit here, e.g.
* Extended [Rate](#rate) with new properties `Type` and `Description`